apps/native-component-list/src/screens/HapticsScreen.tsx

```diff
@@ -1,12 +1,19 @@
 import * as Haptics from 'expo-haptics';
 import React from 'react';
-import { SectionList, StyleSheet, Text, View, ViewStyle } from 'react-native';
+import { SectionList, SectionListData, StyleSheet, Text, View, ViewStyle } from 'react-native';
 
 import Button from '../components/Button';
 import MonoText from '../components/MonoText';
 import Colors from '../constants/Colors';
 
-const sections = [
+type SectionData =
+  | object
```

Comment:
That's incorrect. The actual type we need here is an `empty object type`, namely `{}`. I know we have a lint rule preventing this, but in this case it should be that one 😉
Why? Because of https://mariusschulz.com/articles/the-object-type-in-typescript

Reply:
I thought so too, but figured lint rule may know better. Will fix.